src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -260,7 +260,7 @@ private void SnapToNewSnapshot(IssuesSnapshot newIssues)
 
         public void RequestAnalysis(IAnalyzerOptions options)
         {
-            Provider.RequestAnalysis(FilePath, charset, detectedLanguages, this, ProjectItem, options);
```

Comment:
The `TextBufferIssueTracker` currently still needs the ProjectItem itself (although it should now be easier to refactor this class so that it doesn't).